src/components/MainTodoApp/MainTodoApp.tsx

```diff
@@ -0,0 +1,69 @@
+import { FC } from 'react';
+import { Todo } from '../../types/Todo';
+import { Category } from '../../types/Category';
+import { CompletedTodo } from '../CompletedTodo';
+import { ActiveTodo } from '../ActiveTodo';
+import { LoadingTodo } from '../LoadingTodo';
+
+interface Props {
+  todos: Todo[];
+  category: Category;
+}
+
+export const MainTodoApp: FC<Props> = ({ todos, category }) => {
+  let visibleTodos = todos;
+
+  if (category !== 'all') {
+    visibleTodos = todos.filter(({ completed }) => {
+      return (category === 'completed')
+        ? completed
+        : !completed;
+    });
+  }
+
+  return (
+    <>
+      {visibleTodos.map(({ id, completed, title }) => {
+        return (
+          <section className="todoapp__main" key={id}>
+            {/* This is a completed todo */}
+            {completed && <CompletedTodo title={title} />}
+
+            {/* This todo is not completed */}
+            {!completed && <ActiveTodo title={title} />}
```

Comment:
handle it in the `Todo` component
```suggestion
            <Todo title={title} isCompleted={completed} />
```